### PR TITLE
Streamline Graphql Request Panel

### PR DESCRIPTION
**Pull request by vipulgupta2048**
Solves #23 

I have shifted some information into the Playground as I am fixing it later in another commit. 

Signed-off-by: Vipul Gupta (@vipulgupta2048) <vipulgupta2048@gmail.com>

### CODE REVIEW

**Comment by Nishchit14** [comment]
🚀 Deployed on https://5e8858d9b020059426db8c81--firedoc.netlify.com

**Comment by Nishchit14** [comment]
🚀 Deployed on https://5e88b1b7bb65cdb4f6eb43e7--firedoc.netlify.com

File: source/graphql/impressions/request-panel.mdx [hunk 1]
```diff
@@ -1,37 +1,19 @@
...
 title: 06. Request Panel
 subtitle:
-description: Through this impression we understand more about the Request panel in the GraphQL app of Firecamp
+description: Overview of Request Panel in the Graphql app
 ---
 
 ## Overview
 
-The request panel is made up of playground, collections, explorer and the variables panel on the bottom.
+The `Request Panel` in the GraphQL app is the one-stop hub for crafting, configuring, and saving your queries, subscriptions and mutations. Whether you are debugging your API or integrating 3rd party APIs, you can easily get started with Firecamp.
```
Comment (Nishchit14) [inline source/graphql/impressions/request-panel.mdx:9]: > Whether you are debugging your API or integrating 3rd party APIs, you can easily get started with Firecamp.

This sentence doesn't make any sense in GraphQL doc

File: source/graphql/impressions/request-panel.mdx [hunk 2]
```diff
@@ -1,37 +1,19 @@
...
 
-Request panel is the area where you can craft, tinker and save your queries, subscriptions, and mutations in the GraphQL app. The playground is the base on which we can build your requests, with the help of collections and explorer.
+Through the `Request Panel`, you can get access to several components available in the app. These are as follows:
 
-## Passing Arguments in your queries
+1.  **Playground**
+2.  **Explorer**
+3.  **Collection**
+4.  **Headers**
+5.  **GraphQL Helper Documentation**
```
Comment (Nishchit14) [inline source/graphql/impressions/request-panel.mdx:17]: Change it to `Schema Documentation`

File: source/graphql/impressions/request-panel.mdx [hunk 3]
```diff
@@ -1,37 +1,19 @@
...
-    }
-
-If you have more than one request that is passing parameters in the playground then you can assign specific values for them by just clicking on them and seeing the name change to that specific query down in the variables panel.
-
-## Additional Resources
-
-- Playground
-- Explorer
-- Collections
+When you send a request to your GraphQL endpoint, Firecamp will show the response from the server in the `Response Panel` with the functionality to view, and copy the data received from the server along with other information.
```
Comment (Nishchit14) [inline source/graphql/impressions/request-panel.mdx:19]: > When you send a request to your GraphQL endpoint, Firecamp will show the response from the server in the

when you run a GraphQL query in the playground, You will get a response of that query at the right side `response-panel`.

Comment (Nishchit14) [reply source/graphql/impressions/request-panel.mdx:19]: in graphql, It's query instead of request.

**Comment by Nishchit14** [comment]
@vipulgupta2048 also can we add a screenshot here of request panel mark?

File: source/graphql/impressions/playground.mdx [hunk 4]
```diff
@@ -10,6 +10,28 @@ The Firecamp GraphQL Playground enables you to work with GraphQL APIs to write q
 
 Learn more about how to use the GraphQL playground in through our Actions.
 
+
+## Passing Arguments in your queries
+
```
Comment (Nishchit14) [inline source/graphql/impressions/playground.mdx:15]: By reading this content, It feels like a bit flowless. I could not understand the value of the playground. Here we can simply show how we can write a query and execute them from the playground with a screenshot.

Comment (Nishchit14) [reply source/graphql/impressions/playground.mdx:15]: I think It's been covered in https://github.com/firecampapp/firecamp-doc/pull/32

Comment (vipulgupta2048) [reply source/graphql/impressions/playground.mdx:15]: Yes, this was an impulse decision that I made. I have covered it in 32 pretty well. Let me remove this bit.

**Comment by Nishchit14** [comment]
🚀 Deployed on https://5e8f450e7f21120236e202cb--firedoc.netlify.com

**Comment by Nishchit14** [comment]
🚀 Deployed on https://5e8f45736f27e27e6c320617--firedoc.netlify.com

File: source/graphql/impressions/request-panel.mdx [hunk 5]
```diff
@@ -1,37 +1,19 @@
...
 
-Request panel is the area where you can craft, tinker and save your queries, subscriptions, and mutations in the GraphQL app. The playground is the base on which we can build your requests, with the help of collections and explorer.
+Through the `Request Panel`, you can get access to several components available in the app. These are as follows:
 
-## Passing Arguments in your queries
+1.  **Playground**
+2.  **Explorer**
+3.  **Collection**
+4.  **Headers**
+5.  **Schema Documentation**
```
Comment (Nishchit14) [inline source/graphql/impressions/request-panel.mdx:17]: @vipulgupta2048 Can we add a small description for all? we can put it in the `little things` list.

**Pull request merged**